vignettes/wrapping-apis.Rmd

```diff
@@ -689,6 +723,8 @@ req %>% req_perform()
 
 ## OAuth
 
+JENNY PEDANTIC COMMENT HERE
```

Comment:
I know the whole authentication vs. authorization thing and that we're sort of talking about both. But if we're going to pick one of those words to feature in this paragraph, here authorization somehow feels like the better choice?

Reply:
Good point!